components/esp_lvgl_port/include/esp_lvgl_port_lv_blend.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "esp_log.h"
+#include <sdkconfig.h>
+#include "lvgl.h"
+#include "lv_conf_internal.h"
+
+#if CONFIG_LV_DRAW_SW_ASM_CUSTOM
```

Comment:
In case someone includes this file and the ASM_CUSTOM option is not set, we should at least write a warning that the ASM blender is not actually used

Reply:
You can not include this file using Kconfing without enabling the `CONFIG_LV_DRAW_SW_ASM_CUSTOM`, you can include it just manually. But sure will add a warning.